docs-src/app-dev-context/list-filter.md

```diff
@@ -18,6 +18,22 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **IN**
   - **BETWEEN ... AND**
   - **ORDER BY**
+  - **LIKE**
```

Comment:
This file is also not in use by a guide config - it appears to be a direct duplicate of the concept file "what is a list filter" - so this file too can probably be deleted.